.github/workflows/ci.yaml

```diff
@@ -110,6 +110,9 @@ jobs:
           fetch-depth: 0
           submodules: true
 
+      - name: Install development tools
+        run: sudo apt install -y build-essential pkg-config g++ libx11-dev libxkbfile-dev libsecret-1-dev
```

Comment:
I suspect this was the culprit of the failures: because `npm shrinkwrap` generates the shrinkwrap file from the existing `node_modules` folder (to extract the versions that had been installed), if an optional module failed to install (e.g. because some build dependencies like X11 etc.. were missing) - it wouldn't be in the shrinkwrap file.

Unfortunately, looks like I need some more approvals again to be able to run the workflows and validate this... @jsjoeio @code-asher if you can unblock the run(s) that'd be awesome!